test-cypress/steps/home-steps.js

```diff
@@ -0,0 +1,128 @@
+class HomeSteps {
+
+    static tutorialVisible(shouldBeVisible) {
+        if (shouldBeVisible) {
+            cy.get('.tutorial-container')
+                .should('be.visible');
+        } else {
+            cy.get('.tutorial-container')
+                .should('not.be.visible');
+        }
+    }
+
+    static declineTutorial() {
+        // Click "No, thanks" button
+        cy.get('.tutorial-container .decline-tutorial').click();
+    }
+
+    static selectSPARQLQueryToExecute = function (query) {
+        cy.get('ul.saved-queries li:contains("' + query + '")')
+            .should("be.visible")
+            .trigger('hover')
+            .within(() => {
+                cy.get('.execute-saved-query')
+                    .click({force: true});
+            });
+    };
+
+    static verifyQueryLink(queryName, modifiesRepoModal, queryURL) {
+        HomeSteps.selectSPARQLQueryToExecute(queryName);
+        modifiesRepoModal ? cy.get('.modal-body').should('be.visible') : cy.get('.modal-body').should('not.be.visible');
+        cy.url().should('eq', Cypress.config("baseUrl") + queryURL);
+        cy.visitAndWaitLoader('/');
```

Comment:
What about changing the function to 'verifyQueryLinkAndGoHome'? SInce I execute these steps always one after another. I do not find it necessary to introduce a separate step for this.